frontend/dynamic-demo-plugin/src/utils/telemetry.ts

```diff
@@ -0,0 +1,89 @@
+const SEGMENT_KEY = '';
```

Comment:
Do we plan to eventually make segment its own sample plugin, separate from dynamic-demo-plugin?

Reply:
@spadgett yes we do. This is just an example. For devsandbox we will require creating a separate plugin that can use this example as its foundation.

Maybe what we do is create a repo with a service that can be deployed as a plugin and configured with segment.